jdbc/pom.xml

```diff
@@ -105,6 +105,16 @@
             <artifactId>jcl-over-slf4j</artifactId>
             <version>1.7.13</version>
         </dependency>
+        <dependency>
+            <groupId>org.jdbi</groupId>
```

Comment:
Can you investigate if encryption is supported ? Similar to AES encryption of fields through Hibernate